multigeiger/transmission.cpp

```diff
@@ -165,27 +196,30 @@ void transmit_data(String tube_type, int tube_nbr, unsigned int dt, unsigned int
   #if SEND2DUMMY
   displayStatusLine("Toilet");
   log(INFO, "SENDING TO TOILET ...");
-  send_http_geiger(TOILET, tube_type, dt, hv_pulses, gm_counts, cpm);
+  send_http_geiger(TOILET, tube_type, dt, hv_pulses, gm_counts, cpm, true, XPIN_RADIATION);
   if (have_thp)
-    send_http_thp(TOILET, temperature, humidity, pressure);
+    send_http_thp(TOILET, temperature, humidity, pressure, XPIN_BME280);
   delay(300);
   #endif
 
   #if SEND2MADAVI
   log(INFO, "Sending to Madavi ...");
   displayStatusLine("Madavi");
-  send_http_geiger(MADAVI, tube_type, dt, hv_pulses, gm_counts, cpm);
-  if (have_thp)
-    send_http_thp(MADAVI, temperature, humidity, pressure);
+  if (have_thp) {
+    // send both infos in one request to MADAVI to keep the server load lower
```

Comment:
btw, doing 2 requests is maybe not as bad as one thinks.

if we keep the connection, it is just a 2nd request flowing over existing tcp (and in future tls) connection.

i tried that when experimenting with https, the 2nd+ request is much faster than the first.

Reply:
OK, may be. But Rajko told me, i shall use one request!

Reply:
ok, he told me I SCHOULD use one request, but two are also OK. So I will stay on two calls, code is simpler then. And most of our geigers don‘t have a BME anyway.